packages/components/src/toolbar/stories/index.js

```diff
@@ -1,182 +0,0 @@
-/**
- * WordPress dependencies
- */
-import {
```

Comment:
Was this removal intended?

Reply:
🤦‍♀️ Ah yes... During dev, something happened with my `git` and I noticed `toolbar` may have been affected? But I wasn't able to spot. Thank you for catching this! I'll revert this now